test/src/main/scala/io/finch/test/ServiceSuite.scala

```diff
@@ -3,15 +3,15 @@ package io.finch.test
 import com.twitter.finagle.Service
 import com.twitter.finagle.http.{Request, Response}
 import com.twitter.util.{Await, Duration}
-import org.scalatest.{fixture, Outcome}
+import org.scalatest.{FixtureTestSuite, Outcome}
 
 /**
  * A convenience class that is designed to make it easier to test HTTP services
  * both directly and in integration tests that are served locally. Implementing
  * classes must extend [[org.scalatest.fixture.Suite]] through [[org.scalatest.fixture.FlatSpec]]
```

Comment:
another nitpick but these don't exist anymore.

```suggestion
 * classes must extend [[org.scalatest.FixtureTestSuite]] through [[org.scalatest.flatspec.FixtureAnyFlatSpec]]
```